comixed-frontend/src/assets/i18n/es/comics.json

```diff
@@ -2,283 +2,283 @@
   "comics-effects": {
     "get-scan-types": {
       "error": {
-        "detail": "Failed to load scan types."
+        "detail": "Error al cargar tipos de escaneados."
       }
     },
     "get-formats": {
       "error": {
-        "detail": "Failed to load comic formats."
+        "detail": "Error al cargar formatos de cómics."
       }
     },
     "get-page-types": {
       "error": {
-        "detail": "Failed to load page types."
+        "detail": "Error al cargar tipos de página."
       }
     },
     "get-issue": {
       "error": {
-        "detail": "Failed to load comic id={id}."
+        "detail": "Error al cargar el cómic con id={id}."
       }
     },
     "save-page": {
       "success": {
-        "detail": "Page successfully updated."
+        "detail": "Página actualizada con éxito."
       },
       "error": {
-        "detail": "Failed to save page changes."
+        "detail": "Error al guardar los cambios en la página."
       }
     },
     "set-page-hash-blocking": {
       "success": {
-        "detail": "Successfully {state, select, true{blocked} other{unblocked}} pages with the hash {hash}."
+        "detail": "Páginas con hash {hash} {state, select, true{'bloqueadas'} other{'desbloqueadas'}} correctamente."
```

Comment:
Please remove the single quotes around the translated values here: you only need the text within the curly braces.

Reply:
Solved [2c233768abc812a9e6eea1fcd0bb9a654784f8d7]